jaxtyping/_ipython_extension.py

```diff
@@ -0,0 +1,19 @@
+try:
+    import IPython
+
+    ip = IPython.get_ipython()
+except ImportError:
+    ip = None
+
+from ._import_hook import _JaxtypingTransformer
+
+
+def load_ipython_extension(ip, **kwargs):
+    try:
+        from beartype import beartype as beartype
+    except ImportError:
+        raise ImportError(
+            "Beartype should be installed for jaxtyping IPython extension to work."
+        )
+
+    ip.ast_transformers.append(_JaxtypingTransformer(typechecker="beartype.beartype"))
```

Comment:
I don't think I need to check for existence of IPython here, since if somebody decided to call ```load_ipython_extension directly```, it's not the problem of implementation :) And otherwise, if ```%load_ext``` is used, one will have to have IPython installed, I reckon.

Reply:
SGTM!